tools/test/config/react17/r17-packages.json

```diff
@@ -6,13 +6,19 @@
     "@types/react-dom": "16.9.18",
     "@types/react-is": "17.0.7",
     "@testing-library/react": "12.1.3",
-    "@testing-library/react-hooks": "8.0.1",
-    "@testing-library/user-event": "13.5.0",
```

Comment:
we should probably keep these here, since at least `/react-hooks` will likely be removed soon